examples/sample-library-linking-project/.avail/sample-library-linking/artifact-plans.json

```diff
@@ -0,0 +1,37 @@
+[
+	{
+		"version": "2.0.0.alpha02",
```

Comment:
There are a lot of things with similar numbers.  Not going to block this, but maybe the JSON tag should be "projectVersion" or "artifactVersion" or something like that.

Reply:
This has to be fixed in `avail-artifact`. The comment for `AvailArtifactBuildPlan.version`

```kotlin
* @property version
 *   The version to give to the created artifact. **REQUIRED**
```

So the name change would be `artifactVersion`. As for the name, it could be changed to `artifactVersion`. My only counter to your statement is that the version here is relative to the artifact which is what this JSON object represents. If the version wasn't `2.0.0.alpha02` but instead `1.0.1`, would there be as much confusion? It might be the case that the confusion is caused because we have a lot of alphas flying around right now. I'll just change it to `1.0.0.rc1`.